apps/docs/src/components/HomeLayout/TopNavigation/index.tsx

```diff
@@ -41,7 +41,7 @@ export const TopNavigation: FC<Props> = ({ docs, docsTree }) => {
 					);
 				})}
 			</View>
-			<View>
+			<View style={{ width: styles.logo.width, alignItems: 'flex-end' }}>
```

Comment:
we could create a constant width and the styles share it, like this: 
```typescript
const sharedWidth = 256;
const styles = Stylesheet.create({
  logo: {
    width: sharedWidth,
  },
  button: {
    width: sharedWidth,
  },
})
```

Reply:
Ok, I got it